docs/cache/limits.mdx

```diff
@@ -0,0 +1,145 @@
+---
+sidebar_label: Quotas and Limits
+sidebar_position: 8
+title: Momento Cache service quotas
+pagination_next: null
+description: Examine the namespace and item-level quotas in place with Momento Cache and discover the default and tunable values for your workload based on your support plan.
+hide_title: true
+---
+
+# Namespace and item quotas in Momento Cache
+
+:::info
+Service quotas are directly affected by the support plan on your account. Both <ins>soft and hard limits</ins> are increasable for the Pro tier and Enterprise plans. If you would like to request an increase, please [contact sales](https://www.gomomento.com/contact-us).
+:::
+
+## Namespace and items
+
+<table className="customTable">
+  <thead>
+    <tr>
+      <th>Resource</th>
+      <th>Entity</th>
+      <th>Default</th>
+      <th>Basic support</th>
+      <th>Pro support</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr className="highlightRow">
+      <td colspan="5"><strong>Soft limits</strong></td>
+    </tr>
+    <tr>
+      <td>Operations per second</td>
+      <td>namespace</td>
+      <td>100</td>
+      <td>1,000</td>
+      <td>10,000</td>
+    </tr>
+    <tr>
+      <td>Hot storage</td>
+      <td>namespace</td>
+      <td>100 MB</td>
+      <td>1 GB</td>
+      <td>10 GB</td>
+    </tr>
+    <tr className="highlightRow">
+      <td colspan="5"><strong>Hard limits</strong></td>
+    </tr>
+    <tr>
+      <td>Time to live (TTL)</td>
+      <td>item</td>
+      <td colspan="3">24 hours</td>
+    </tr>
+    <tr>
+      <td>Item size</td>
+      <td>item</td>
+      <td colspan="3">5 MB</td>
```

Comment:
Hmm the default limit is 1MB afaik. We should probably add that in the soft limit section too for the sake of completeness.